meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf_%.bbappend

```diff
@@ -18,3 +18,9 @@ do_deploy:append:mx8-nxp-bsp() {
     install -m 0644 ${S}/build-optee/${ATF_PLATFORM}/release/bl31.bin ${DEPLOYDIR}/arm-trusted-firmware.bin
     install -m 0644 ${S}/build-optee/${ATF_PLATFORM}/release/bl31/bl31.elf ${DEPLOYDIR}/arm-trusted-firmware.elf
 }
+
+#FIXME: build -> build-optee
```

Comment:
Could you please add notes on what it means?
I would describe why we deploy non-optee versions here and in a commit message to not miss this later.